### ISSUE TITLE
Layout/ClosingHeredocIndentation and <<

### ISSUE THREAD

**Issue by bquorning** [rubocop-hq/rubocop#5968]
## Expected behavior

```rb
def foo
  <<STR
bar
STR
end
```

No offense should be detected here, since `<<` heredocs *must* end with an *unindented* identifier (here `STR`). Only `<<-` or `<<~` heredocs allow indentation of the closing indentifier.

## Actual behavior

One offense is found:
```
C: Layout/ClosingHeredocIndentation: STR is not aligned with <<STR.
STR
^^^
```

## Steps to reproduce the problem

Run `rubocop --only Layout/ClosingHeredocIndentation` on a file with the above contents.

## RuboCop version

Include the output of `rubocop -V`. Here's an example:

```
$ rubocop -V
0.57.1 (using Parser 2.5.1.0, running on ruby 2.5.1 x86_64-darwin15)
```

**Comment by dvandersluis** [comment]
I added a PR to have Layout/ClosingHeredocIndentation skip `<<` heredocs.
